db-engine/src/main/java/com/splicemachine/db/iapi/types/UserType.java

```diff
@@ -249,7 +254,27 @@ public Object getObject()
 	{
 		return value;
 	}
-		
+
+	@Override
+	public Object getSparkObject() throws StandardException {
+		ByteArrayOutputStream bos = new ByteArrayOutputStream();
+		ObjectOutput out = null;
+		try {
+			out = new ObjectOutputStream(bos);
+			out.writeObject(value);
+			out.flush();
+			return bos.toByteArray();
+		} catch (IOException e) {
+			throw new RuntimeException(e);
+		} finally {
+			try {
+				bos.close();
+			} catch (IOException ex) {
+				// ignore close exception
+			}
+		}
+	}
```

Comment:
How about making use of a try-with-resources block to simplify things a bit?
```java
	@Override
	public Object getSparkObject() throws StandardException {
		try (ByteArrayOutputStream bos = new ByteArrayOutputStream()) {
			ObjectOutput out = new ObjectOutputStream(bos);
			out.writeObject(value);
			out.flush();
			return bos.toByteArray();
		} catch (IOException e) {
			throw new RuntimeException(e);
		}
	}
```
Also, why are you throwing `RuntimeException` instead of `StandardException`?

Reply:
@chuckwondo good points, I simplified it by using SerializationUtils.serialize()/deserialize() as used elsewhere on the class.